pulp_rpm/tests/functional/constants.py

```diff
@@ -32,13 +32,22 @@
 RPM_UNSIGNED_FIXTURE_URL = urljoin(PULP_FIXTURES_BASE_URL, 'rpm-unsigned/')
 """The URL to a repository with unsigned RPM packages."""
 
-RPM_FIXTURE_COUNT = 39  # 35 Packages + 4 UpdateRecord units
+RPM_PACKAGES_FEED_COUNT = 35
+"""The number of packages available at
+:data:`RPM_SIGNED_FIXTURE_URL` and :data:`RPM_UNSIGNED_FIXTURE_URL`
+"""
+
+RPM_UPDATED_RECORDED_COUNT = 4
+"""The number of updated record units."""
+
+RPM_FIXTURE_COUNT = RPM_PACKAGES_FEED_COUNT + RPM_UPDATED_RECORDED_COUNT
```

Comment:
I don't think this constant is needed anymore, it's a holdover from when everything was mixed together.